apps/agent/src/mastra/workflows/github-notify-reviewers.ts

```diff
@@ -0,0 +1,177 @@
+import { Step, Workflow } from '@mastra/core/workflows'
+import { z } from 'zod'
+import { discordClient } from '../../lib/discord'
+import { GITHUB_REPO, githubClient } from '../../lib/github'
+import { PullRequest } from '../../lib/type'
+import { formatDate } from '../../utils/datetime'
+import { DISCORD_GITHUB_MAP } from '../../constants/discord'
+
+class NotifyReviewersWorkflow {
+  private workflow: Workflow
+
+  constructor() {
+    this.workflow = new Workflow({
+      name: 'Notify Reviewers',
+    })
+  }
+
+  private getPendingReviews = new Step({
+    id: 'get-pending-reviews',
+    execute: async () => {
+      // get all PR
+      const prs = await githubClient.getRepoPRs(GITHUB_REPO, {
+        isMerged: false,
+        isOpen: true,
+      })
+
+      // get PRs with reviewer assigned but not reviewed yet
+      const todayPRsWithReviewerAssigned = await Promise.all(
```

Comment:
This query isn't just for today's PRs, right? If that's true, please change the name